apps/src/code-studio/pd/foorm/foormHelpers.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Creates a map of entity name to entity version where the version is the highest for that name
```

Comment:
Great to add this, I bet we can eventually move some more stuff that's shared between the non-generic components in here as well.